v2/cmd/wails/internal/commands/initialise/templates/templates/svelte/frontend/package-lock.tmpl.json

```diff
@@ -1,3638 +1,1016 @@
 {
   "name": "{{.ProjectName}}",
-  "version": "1.0.0",
-  "lockfileVersion": 1,
+  "version": "0.0.0",
```

Comment:
I'm not sure we need this file? People might not trust it.

Reply:
This should be decided by the user, and some projects will use the front-end part as a sub-module (for example: https://github.com/letheanVPN/desktop). Front-end developers and back-end developers maintain it separately, I think it exists If necessary, I usually modify it myself to add version tags.